tasks/database.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Database
+  PATH_FILE = 'database.yaml'.to_s
+
+  def save_data
+    data = {
+      authors: @authors,
+      books: @books,
+      readers: @readers,
+      orders: @orders
+    }
+
+    File.write(PATH_FILE, data.to_yaml)
+  end
+
+  def load_data
+    data = File.exist?(PATH_FILE) ? load_library_from_seeds : generate_data
+    add_entity_to_library(data)
+  end
+
```

Comment:
`load_library_from_seeds`, `generate_data`, `add_entity_to_library` should be private methods

Reply:
done